tools/pd-ctl/pdctl/command/config_command.go

```diff
@@ -386,6 +389,30 @@ func postConfigDataWithPath(cmd *cobra.Command, key, value, path string) error {
 	return nil
 }
 
+func checkMaxReplicas(cmd *cobra.Command, value any) {
+	newReplica, ok := value.(float64)
```

Comment:
seem line 372 already checked it?

Reply:
If err is not nil, val will be the type of string.

Reply:
Is that correct?